fastlane/Fastfile

```diff
@@ -662,18 +661,18 @@ DESC
   private_lane :generate_snapshots do |options|
     require 'rest-client'
 
+    # Get CircleCI token
+    circle_token = ENV["CIRCLE_TOKEN"]
+    UI.user_error! "Please set the CIRCLE_TOKEN environment variable" unless circle_token
+
     # Prompt branch
     default_branch = git_branch
     branch = UI.input("Branch (defaults to #{default_branch}): ")
     branch = default_branch if branch == ""
 
-    # Get CircleCI token
-    circle_token = ENV["CIRCLE_TOKEN"]
-    UI.user_error! "Please set the CIRCLE_TOKEN environment variable" unless circle_token
-
     # Make request
     headers = {"Circle-Token": circle_token, "Content-Type": "application/json", "Accept": "application/json"}
-    data = {parameters: {options[:pipeline] => true}, branch: branch}
+    data = {parameters: {:action => options[:action]}, branch: branch}
```

Comment:
the `pipeline: true` way didn't work. I changed it to call the action, which is what we use for release trains